squidb-tests/src/com/yahoo/squidb/test/TriggerTesterSpec.java

```diff
@@ -16,4 +16,8 @@
 
     @ColumnSpec(defaultValue = "0")
     int value2;
+
+    String str1;
+
+    String str2;
```

Comment:
Eh, let's just leave it. We use value1 and value2 for some numeric things in the trigger and I'd rather not update the rest of the test.